### DISCUSSION TITLE
TabView underline color

### DISCUSSION THREAD

**Discussion by XamMattia83** [enisn/UraniumUI#805] [Q&A] [answered]
Hello!

i'd like to know if it's possible  and how to change the Tabview Selected underline color. I'm using my colors and styles so i'm overriding  Uranium  ones.  Thanks in advance

**Comment by malsabi** [comment, marked as answer]
@XamMattia83 The Uranium UI TabView uses a custom template, and the underline color is based on the primary color. If you override the colors and use the same names "Primary" and "PrimaryDark" with your preferred colors, you'll see the selection color change accordingly.

To learn more about which other colors the custom template uses, you can check the source code here:
https://github.com/enisn/UraniumUI/blob/develop/src/UraniumUI.Material/Controls/TabView.cs

You can also create your own custom template for the header.

**Comment by XamMattia83** [comment]
Got it ;) thank you...perfect.

I placed the question because i had a strange underline "black"  while my Primary is Blue. Looking at the source code i see that underline is made by a BoxView...and here the issue:  the default MAUI template set a default Style for Boxview with a background color == Gray950  (light theme) so i had to remove the style to achieve the expected behavior ;)
